scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterParam.json

```diff
@@ -1,73 +1,93 @@
-          "uucp"
-        ]
-      },
-      "resourceTagValues": {
-        "value": {
-          "<existingOrnew-tag-name1>": "<existingOrnew-tag-value1>",
-          "<existingOrnew-tag-name2>": "<existingOrnew-tag-value2>",
-          "<existingOrnew-tag-nameN>": "<existingOrnew-tag-valueN>"
-        }
-      },
-      "dataCollectionInterval":  {
-        "value" : "1m"
-      },
-      "namespaceFilteringModeForDataCollection": {
-        "value": "Off"
-      },
-      "namespacesForDataCollection": {
-        "value": [ "kube-system", "gatekeeper-system", "azure-arc" ]
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "aksResourceId": {
+      "value": "/subscriptions/<SubscriptionId>/resourcegroups/<ResourceGroup>/providers/Microsoft.ContainerService/managedClusters/<ResourceName>"
+    },
+    "aksResourceLocation": {
+      "value": "<aksClusterLocation>"
+    },
+    "workspaceResourceId": {
+      "value": "/subscriptions/<SubscriptionId>/resourceGroups/<ResourceGroup>/providers/Microsoft.OperationalInsights/workspaces/<workspaceName>"
+    },
+    "workspaceRegion": {
+      "value": "<workspaceRegion>"
+    },
+    "enableSyslog": {
+      "value": false
+    },
+    "syslogLevels": {
+      "value": [
+        "Debug",
+        "Info",
+        "Notice",
+        "Warning",
+        "Error",
+        "Critical",
+        "Alert",
+        "Emergency"
+      ]
+    },
+    "syslogFacilities": {
+      "value": [
+        "auth",
+        "authpriv",
+        "cron",
+        "daemon",
+        "mark",
+        "kern",
+        "local0",
+        "local1",
+        "local2",
+        "local3",
+        "local4",
+        "local5",
+        "local6",
+        "local7",
+        "lpr",
+        "mail",
+        "news",
+        "syslog",
+        "user",
+        "uucp"
+      ]
+    },
+    "resourceTagValues": {
+      "value": {
+        "<existingOrnew-tag-name1>": "<existingOrnew-tag-value1>",
+        "<existingOrnew-tag-name2>": "<existingOrnew-tag-value2>",
+        "<existingOrnew-tag-nameN>": "<existingOrnew-tag-valueN>"
       }
+    },
+    "dataCollectionInterval": {
+      "value": "1m"
+    },
+    "namespaceFilteringModeForDataCollection": {
+      "value": "Off"
+    },
+    "namespacesForDataCollection": {
+      "value": [
+        "kube-system",
+        "gatekeeper-system",
+        "azure-arc"
+      ]
+    },
+    "defaultStreams": {
+      "value": [
+        "Microsoft-ContainerLog",
+        "Microsoft-ContainerLogV2",
+        "Microsoft-KubeEvents",
+        "Microsoft-KubePodInventory",
+        "Microsoft-KubeNodeInventory",
+        "Microsoft-KubePVInventory",
+        "Microsoft-KubeServices",
+        "Microsoft-KubeMonAgentEvents",
+        "Microsoft-InsightsMetrics",
+        "Microsoft-ContainerInventory",
+        "Microsoft-ContainerNodeInventory",
+        "Microsoft-Perf"
+      ]
```

Comment:
how about ContainerImageInventory, ContainerServiceLog, Heartbeat, and syslog for syslog

Reply:
> how about ContainerImageInventory, ContainerServiceLog, Heartbeat, and syslog for syslog
>  1. We dont support ContainerImageInventory & ContainerServiceLog 2. Heartbeat part of the MDSD and its not CI stream 3. Syslog configured outside of the ContainerInsightsExtension DCR. Refer ARM template to understand more on this.